### PR TITLE
Update url.md

### PR DESCRIPTION
**Pull request by marble-sh**
- [ ] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [x] `brew audit` is error-free.
- [x] `brew style` reports no offenses.

Sister pull request to https://github.com/Homebrew/brew/pull/10548

### CODE REVIEW

File: doc/cask_language_reference/stanzas/url.md [hunk 1]
```diff
@@ -116,4 +116,11 @@ Example for using the block syntax: [vlc-nightly.rb](https://github.com/Homebrew
 
 In rare cases, you might need to set URL parameters like `cookies` or `referer` while also using the block syntax.
 
-This is possible by returning a two-element array as a block result. The first element of the array must be the download URL; the second element must be a `Hash` containing the parameters.
+This is possible by passing the options as part of the initial block syntax.
+
+
+```rb
+url referer: "https://example.com", using: MyDownloadStrategy do
+  ...
```
Comment (vitorgalvao) [inline doc/cask_language_reference/stanzas/url.md:124]: We’d rather have a real example. We don’t do the ellipses on examples anywhere else. Also, `MyDownloadStrategy` will be confusing. It’s best to have one specific real example people can adapt than an example that is too generic.

Comment (marblenix) [reply doc/cask_language_reference/stanzas/url.md:124]: Would it be too much to use my own repository as an example? As far as I know I'm the only one who uses both `url do` blocks and download options.

Comment (vitorgalvao) [reply doc/cask_language_reference/stanzas/url.md:124]: > Would it be too much to use my own repository as an example?

Go ahead!

**Comment by marblenix** [comment]
closing due to alternate fix in original pull request

**Pull request closed without merging**